lib/src/data_frame/data_frame_impl.dart

```diff
@@ -185,14 +182,26 @@ class DataFrameImpl with SerializableMixin implements DataFrame {
 
   DataFrame _dropByNames(Iterable<String> names, Iterable<Series> series) {
     final uniqueNames = Set<String>.from(names);
-    final newSeries = series
-        .where((series) => !uniqueNames.contains(series.name));
+    final newSeries =
+        series.where((series) => !uniqueNames.contains(series.name));
 
     return DataFrame.fromSeries(newSeries);
   }
 
-  Map<String, Series> _getCachedOrCreateSeriesByName() =>
-      _seriesByName ??= Map
-          .fromEntries(series.map((series) => MapEntry(series.name, series)));
+  Map<String, Series> _getCachedOrCreateSeriesByName() => _seriesByName ??=
+      Map.fromEntries(series.map((series) => MapEntry(series.name, series)));
   Map<String, Series>? _seriesByName;
+
+  @override
+  String toString({int maxRows = 10, int maxCols = 7}) {
+    return dataFrameToString(
+      this,
+      maxRows: maxRows,
+      maxCols: maxCols,
+    );
+  }
+
+  String visua() {
```

Comment:
@tadeohepperle apparently, you forgot to remove this method. Could you please remove it?

Reply:
@gyrdym Ah yes I did. Sorry.